package.json

```
@@ -7,7 +7,7 @@
   },
   "description": "Example projects for the freeCodeCamp.org curriculum",
   "scripts": {
-    "lint": "eslint apps --max-warnings 0",
+    "lint": "eslint src test --max-warnings 0 && prettier src test --check",
```

```suggestion
    "lint": "eslint apps test --max-warnings 0 && prettier apps test --check",
```

Entirely my fault. I'm too used to typescript apps.